lib/utils.js

```diff
@@ -306,4 +306,22 @@ export default {
 		const { vendorDependencies } = await fs.readJSON(packageJSON);
 		return vendorDependencies.node;
 	},
+
+	/**
+	 * Retrieve a setting value, handling falling back to the old namespace
+	 *
+	 * @param {string} configString - The setting string to retrieve
+	 * @returns {string|boolean|number}
+	 */
+	getConfigSetting (configString) {
+		configString = configString.replace(/^(appcelerator-titanium|titanium)\./, '');
```

Comment:
Handling `appcelerator-titanium` shouldn't be necessary as I think I got every instance of it, but I figured it was best to be safe